frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -388,14 +438,15 @@ describe('NewRunParametersV2', () => {
       handlePipelineRootChange: jest.fn(),
       handleParameterChange: handleParameterChangeSpy,
     };
-    render(<NewRunParametersV2 {...props}></NewRunParametersV2>);
+    render(<NewRunParametersV2 {...props} />);
 
     const structParam = screen.getByLabelText('structParam - STRUCT');
     fireEvent.change(structParam, { target: { value: '{"A":1,"B":2}' } });
     expect(handleParameterChangeSpy).toHaveBeenCalledTimes(1);
     expect(handleParameterChangeSpy).toHaveBeenLastCalledWith({
       structParam: { A: 1, B: 2 },
     });
+    expect(screen.getByDisplayValue('{"A":1,"B":2}'));
```

Comment:
NIT: I think `expect(` here can be omitted. Because the test will fail if `screen.getByDisplayValue('{"A":1,"B":2}')` cannot find anything.